rebar.config

```diff
@@ -22,24 +22,24 @@
 {escript_name, etylizer}.
 {escript_emu_args, "%%! -escript main etylizer_main -pz etylizer/etylizer/ebin\n"}.
 
-{erl_opts, [no_debug_info, {i, "src"}]}.
+{erl_opts, [{i, "src"}, debug_info]}.
 
 {profiles,[
   {test, [
-    {erl_opts, [{d, debug}, debug_info]},
-    {deps, [{proper, "1.4.0"}, coveralls]},
+    %{erl_opts, [{d, debug}, debug_info]},
+    {deps, [{proper, "1.5.0"}, coveralls]},
     {extra_src_dirs, [{"test", [{recursive, true}]}]}
   ]},
   {default, [
-    {erl_opts, [{d, debug}, debug_info]},
+    % {erl_opts, [{d, debug}, debug_info]},
     {extra_src_dirs, [{"test", [{recursive, true}]}]}
   ]},
   {prod, [
     {erl_opts, [no_debug_info]}
   ]}
 ]}.
 
-{cover_enabled, true}.
-{cover_export_enabled, true}.
+{cover_enabled, false}.
```

Comment:
Enabling cover results in everything rebuilding and increases test time by a lot!